src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -0,0 +1,208 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, FieldArray } from 'redux-form';
+import moment from 'moment';
+
+import { Collapsible } from '@edx/paragon';
+import RenderInputTextField from '../RenderInputTextField';
+import RenderSelectField from '../RenderSelectField';
+import TranscriptLanguage from './TranscriptLanguage';
+
+const formatCourseRunTitle = (courseRun) => {
+  if (courseRun) {
+    const startDateString = moment(courseRun.start).format('MMM Do YYYY');
+    const pacingTypeString = courseRun.pacing_type.split('_').map(pacingType =>
+      pacingType.charAt(0).toUpperCase() + pacingType.slice(1)).join(' ');
+    return (
+      <div className="course-run-label">
+        <span>{`${startDateString} - ${pacingTypeString}`}</span>
+        <div className="course-run-studio-url">
+          {'Studio URL - '}
+          <a href={`${process.env.STUDIO_BASE_URL}/course/${courseRun.key}`}>
+            {`${courseRun.key}`}
+          </a>
+        </div>
+        <div className="course-run-preview-url">
+          {'Preview URL - '}
+          <a href={`${courseRun.preview_url}`}>
+            {'View about page'}
+          </a>
+        </div>
+      </div>
+    );
+  }
+  return (
+    <div>
+      <span>Your new course run</span>
+    </div>
+  );
+};
+
+class CollapsibleCourseRunFields extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleRemove = this.handleRemove.bind(this);
+  }
+
+  handleRemove(index) {
+    this.props.fields.remove(index);
+  }
+
+  render() {
+    const { fields, courseRuns, languageOptions } = this.props;
+    return (
+      <div>
+        {fields.map((courseRun, index) => (
+          <Collapsible
```

Comment:
For a11y you should add an `iconId` attribute to the collapsible that is just something like `collapsible-icon-${index}` to avoid an issue with paragon's dumb id duplication

Reply:
Ooooh nice! Thanks! And we're cool with the index being included in the unique id? I think our linter typically yells at us for that

Reply:
Yeah, you shouldn't use index for the `key`, but it's fine for this